dotnet/Logging/Logger.cs

```diff
@@ -7,14 +7,21 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Runtime.InteropServices;
+using WinSCP.Internal;
 
 namespace WinSCP
 {
     internal class Logger : IDisposable

```

Comment:
note: maybe rename of Logger would be better, but for simplicity I tried to make the changes a small as possible